Services/PizzaService.cs

```diff
@@ -0,0 +1,97 @@
+using ContosoPizza.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ContosoPizza.Services
+{
+    /// <summary>
+    /// Pizza Service.
+    /// </summary>
+    public static class PizzaService
```

Comment:
Can we impalement the PizzaServcie class with interface and we can use dependency injection to inject the dependencies in other classes.